### PR TITLE
Orizaba is default Eiffel protocol

### PR DESCRIPTION
**Pull request by z-sztrom**
### Applicable Issues
Fixes https://github.com/eiffel-community/eiffel-remrem/issues/14

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:  Roman Szturc roman.szturc.ext@ericsson.com

### CODE REVIEW

File: publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java [hunk 1]
```diff
@@ -44,19 +48,20 @@ public class EventTemplateHandler {
...
+        .jsonProvider(new JacksonJsonNodeJsonProvider(mapper))
+        .mappingProvider(new JacksonMappingProvider(mapper))
+        .build();
 
     // eventTemplateParser
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     public JsonNode eventTemplateParser(String jsonData , String eventName){
         JsonNode updatedJson = null;
-        JsonFactory factory = new JsonFactory();
-        ObjectMapper mapper = new ObjectMapper(factory);
```
Comment (jainadc9) [inline publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java:56]: Why to refactor its same as old code

Comment (jainadc9) [reply publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java:56]: what about NON_EMPTY ?

Comment (z-sztrom) [reply publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java:56]: > Why to refactor its same as old code

mapper is instance variable now, used to create/initialize configuration variable, used in several different methods. That's why it changed from local var to instance var.

Comment (z-sztrom) [reply publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java:56]: > what about NON_EMPTY ?

see line 54

**Review by jainadc9 — APPROVED** [review]

File: publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java (hunk 1 above)
Comment (z-sztrom) [reply publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/EventTemplateHandler.java:56]: Without this change tests fail with errors like this:

```
[ERROR]   EventTemplateHandlerTest.testEventParserWithEventEiffelTestSuiteFinishedEvent:234->testParser:369 eventParams.links[target=e269b37d-17a1-4a10-aafb-c108735ee48c]
Unexpected: domainId
 ; eventParams.links[target=e269b37d-17a1-4a10-aafb-c108735ee48d]
Unexpected: domainId

[ERROR] Tests run: 50, Failures: 28, Errors: 0, Skipped: 0
```

**Pull request merged**